flagsmith-core.ts

```diff
@@ -393,6 +399,27 @@ const Flagsmith = class {
                 }
             }
 
+            const migrateAsyncStorageCache = async () => {
```

Comment:
We can remove this as a result of our conversation right ?

Reply:
Of course, I hadn't noticed that this code was in my last commit, thanks